docs/variants/protectli_vp2420/hardware-matrix.md

```diff
@@ -0,0 +1,35 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configuration used for validation of the
+coreboot port on the Protectli VP2420 firewall.
+
+## Protectli VP2420
+
+| Component              | Description                                              |
+|------------------------|----------------------------------------------------------|
+| **CPU**                | Intel(R) Celeron(R) J6412 @ 2.60GHz                      |
+| **RAM**                | CRUCIAL CT4G4SFS824A                                     |
+|                        | SAMSUNG M471A4G43MB1-CTD                                 |
+|                        | SAMSUNG M471A1K43CB1-CTD                                 |
+|                        | SAMSUNG M471A5244BB0-CRC                                 |
+|                        | KINGSTON KVR24S17S8/8                                    |
+|                        | KINGSTON KVR26S19S8/16                                   |
+| **Flash memory**       |                                                          |
+| **SSD**                | SSD Samsung 860 EVO M.2 NZ-N6E250 250GB                  |
+| **MMC drive**          | SAMSUNG 8GTF4R (on-board)                                |
+| **USB pendrives**      | 1. SanDisk USB 3.2Gen1 16 GB                             |
+|                        | 2. SanDisk USB 3.2Gen1 16 GB                             |
+|                        | 3. USB Type-C Hub Pro UCN3286                            |
+| **Attached devices**   | 1. Logitech, Inc. Keyboard K120                          |
+|                        | 2. Dell Mouse MS116p                                     |
+|                        | 3. USB Type-C Hub Pro UCN3286                            |
+| **Wireless card**      | Intel Wi-Fi 6 AX200                                      |
+| **Display**            | HDMI 1920x1080p, DP 1920x1080p                           |
+| **Ethernet**           | 4x intel i225 (on-board)                                 |
+| **TPM**                | PC Engines TPM1A LPC TPM                                 |
```

Comment:
Are we sure about that?
We moved to Protectli TPMs already

Reply:
@miczyg1 I have one additional TPM module. We might insert it in VP2420 platform, which is used for developing and testing

Reply:
Changed.

Reply:
Ohh wait... The current Protectli TPMs use LPC interface, while the platform has a SPI TPM header... I guess we have to leave this field empty for now...

Reply:
Sure, removed it.